apps/screloc/main.cpp

```diff
@@ -270,35 +273,56 @@ class Reloc : public Client::Application {
 						}
 					}
 
+					bool relocated = false;
 					try {
 						org = process(org.get());
+						relocated = true;
 					}
 					catch ( std::exception &e ) {
 						SEISCOMP_ERROR("  + processing failed - %s", e.what());
+						relocated = false;
+					}
+					if ( !org ) { // safety belt, but it should not happen
```

Comment:
As you do continue with `org == nullptr`, this case must be handled in all other lines where `org` is being accessed.

Reply:
I totally missed that, thanks.